layouts/partials/sidebar.html

```diff
@@ -22,28 +22,16 @@
     {{$parentItem := . }}
       {{ if and (.HasChildren) (eq $currentSection $parentItem.Identifier) }}
         <aside id="main-sidebar">
-        {{ range .Children }}
-          {{ if .HasChildren }}
```

Comment:
I removed the `{{ if .HasChildren }}` block since we don't have children in this case. I guess we can figure it out later?